### PR TITLE
Update childrens day copy

### PR DESCRIPTION
**Pull request by MissPhilbin**
Update copy of activities on the children's day page.

### CODE REVIEW

File: pages/children.md [hunk 1]
```diff
@@ -10,10 +10,9 @@ Our day for children is back!
 We're hosting a fun-packed day of activities and workshops for children to
 learn all about getting the most out of the world with Python.
 
-We're working with the Raspberry Pi Foundation, and some of the team who
-brought Python to the Micro:bit will be coming along to show you how use Python
-to write games.  There'll be a chance to play with robots, and to learn how to
-program Minecraft with Python!
+This year we're working with the Raspberry Pi Foundation, Code Club and some of the team who
+brought Python to the BBC Micro:bit. They will be coming along to show you how use Python
```
Comment (alexwlchan) [inline pages/children.md]: Nitpick: the BBC computer is stylised either "Micro Bit" or "micro:bit".

Also, I missed this in the original: should it be "how _to_ use Python to write"?

File: pages/children.md [hunk 2]
```diff
@@ -49,8 +50,8 @@ control Minecraft! Fun for kids and adults.
 
 ### Python for Parents
 
-Because you have no idea what just happened but now your kids are enthusiastic
-and keep asking questions...
+That's right, a session for parent's of young coders because you have no idea what just happened but now your kids are enthusiastic
```
Comment (alexwlchan) [inline pages/children.md]: Nitpick: Apostrophe on "parent's" should be removed.

File: pages/children.md (hunk 1 above)
Comment (MissPhilbin) [reply pages/children.md]: Any chance you can just amend the errors? It's not clear what your process is for copy edits, merges and pushes to the website, but this info needs to go up asap. Thanks.

Comment (alexwlchan) [reply pages/children.md]: Unfortunately I don't have write access to the repo (yet), so I can't do anything directly here.

Happy to do so if somebody grants me write access (I know @inglesp mentioned this yesterday?).

**Pull request closed without merging**